src/components/reusable/overflowMenu/overflowMenu.stories.js

```diff
@@ -29,6 +28,7 @@ const args = {
   verticalDots: false,
   fixed: false,
   assistiveText: 'Toggle Menu',
+  description: 'Destructive Action',
```

Comment:
Maybe don't use this here. It's a bit confusing because it's a prop for the sub-component.